### PR TITLE
ALP manifest update for Istio v1.7 and v1.6 support

### PR DESCRIPTION
**Pull request by realgaurav**
## Description

ALP manifest update for Istio v1.7(`v1.7.4`) and v1.6(`v1.6.13`) support.
Adjust `securityContext` in dikastes manifest.

## Todos

- [x] Tests
- [x] Release Note

## Release Note

```release-note
Updated ALP documentation to support recent Istio versions (v1.7 and v1.6).
```

### CODE REVIEW

**Review by mgleung — APPROVED** [review]
LGTM

**Comment by caseydavenport** [comment]
I think this should have a release note - @realgaurav could you add one?

File: security/app-layer-policy.md [hunk 1]
```diff
@@ -82,23 +124,59 @@ The sidecar injector automatically modifies pods as they are created to work wit
...
 
+{% tabs %}
+<label:Istio v1.7.x,active:true>
+<%
+```bash
+curl {{ "/manifests/alp/istio-inject-configmap-1.7.yaml" | absolute_url }} -o istio-inject-configmap.yaml
+kubectl patch configmap -n istio-system istio-sidecar-injector --patch "$(cat istio-inject-configmap.yaml)"
+```
+
+[View sample manifest]({{ "/manifests/alp/istio-inject-configmap-1.7.4.yaml" | absolute_url }}){:target="_blank"}
```
Comment (mgleung) [inline security/app-layer-policy.md:135]: I think this is supposed to be `istio-inject-configmap-1.7`

**Review by mgleung — APPROVED** [review]
LGTM

**Pull request merged**